lib/index.js

```diff
@@ -22,7 +22,7 @@ var Engine = CoreObject.extend({
     }
 
     if (!this.engine) {
-      throw new Error('Can not fine engine to use for migration!');
+      throw new Error('Can not find engine to use for migration!');
```

Comment:
i also just came across https://en.oxforddictionaries.com/usage/cannot-or-can-not :)